oracle/controllers/instancecontroller/instance_controller.go

```diff
@@ -507,15 +511,15 @@ func (r *InstanceReconciler) reconcileDatabaseInstance(ctx context.Context, inst
 }
 
 // enableDnfs enables dNFS protocol in Oracle database.
-func (r *InstanceReconciler) enableDnfs(ctx context.Context, inst v1alpha1.Instance) error {
+func (r *InstanceReconciler) enableDnfs(ctx context.Context, inst v1alpha1.Instance, enable bool) error {
```

Comment:
nit: rename to setDnfs as it supports both enable and disable DNFS now :)

Reply:
Its fixed in https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/pull/181/commits, deleting this PR in favor of that.